api/server.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import json
+from os import path
+from flask import Flask
+from flask_restful import Resource, Api, abort, reqparse
+
+app = Flask(__name__)
+api = Api(app)
+
+parser = reqparse.RequestParser()
+parser.add_argument('limit', type=int, required=False,
+                    location='args')
+parser.add_argument('offset', type=int, required=False, location='args')
+
+
+def get_path(filename):
+    return 'csv/' + filename + '.csv'
+
+
+def csv_exists(filename):
+    return path.isfile(get_path(filename))
+
+
+def read_csv(filename, limit=-1, offset=0):
+    limit = limit or -1
+    offset = offset or 0
```

Comment:
Using `or` as a null-fallback mechanism for non-boolean values is a bad practise as it can lead to unintended consequences. E.g.:
```Python
def fun(min_items=None):
  min_items = min_items or 5
```
In this case, there is no way to pass 0 as argument as it would fall back to 5, because 0 is falsy.

A better and more explicit way to do such a fallback is to use the inline if-else, even if it is more verbose:
```Python
min_items = min_items if min_items is not None else 5
```